src/Controls/src/Core/ContentPresenter.cs

```diff
@@ -131,4 +130,5 @@ Size ICrossPlatformLayout.CrossPlatformArrange(Rect bounds)
 			return bounds.Size;
 		}
 	}
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
do we need this around the entire contents of the file? 

Can we just put the pragma disable and pragma restore around the specific parts of the code where this needs to be disabled?